libc/config/darwin/arm/entrypoints.txt

```diff
@@ -94,6 +94,9 @@ set(TARGET_LIBC_ENTRYPOINTS
     libc.src.stdlib.calloc
     libc.src.stdlib.realloc
     libc.src.stdlib.free
+
+    libc.src.stdio.vsprintf
```

Comment:
Add `# stdio.h external entrypoints` comment for consistency, and add this  to the other targets that support `sprintf` and `malloc`.